src/Microsoft.ML.TimeSeries/SsaAnomalyDetectionBase.cs

```diff
@@ -201,6 +209,47 @@ public sealed class State : AnomalyDetectionStateBase
             private SequenceModelerBase<Single, Single> _model;
             private SsaAnomalyDetectionBase _parentAnomalyDetector;
 
+            public State()
+            {
+
+            }
+
+            public State(ModelLoadContext ctx) : base(ctx)
+            {
+                WindowedBuffer = new FixedSizeQueue<float>(
+                    ctx.Reader.ReadInt32(), ctx.Reader.ReadInt32(), ctx.Reader.ReadSingleArray());
```

Comment:
>ctx.Reader.ReadInt32(), [](start = 20, length = 23)

When you write the utility, please make sure to do proper validation of the data you're reading from disk.